boards/peregrine/sam4l_wm400_cape/sam4l_wm400_cape_defconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2020-2024 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_CORTEX_M_SYSTICK=y
+
+CONFIG_USE_DT_CODE_PARTITION=y
```

Comment:
If this is essentially and required for all builds then it should be force enabled by soc (or board) Kconfig

Reply:
done. This board uses a SAM-BA bootloader in flash and require this option to relocate code to the correct place.